R/paren_body_linter.R

```diff
@@ -0,0 +1,33 @@
+#' @describeIn linters check that there is a space between the
+#' right parenthesis and the function body when no braces are used
+#' to define a function.
+#'
+#' @export
+paren_body_linter <- function() {
+  Linter(function(source_file) {
+    if (is.null(source_file$xml_parsed_content)) return(NULL)
+    
+    xpath <- paste(
+      "//expr[",
+      "@line1 = preceding-sibling::FUNCTION/@line1",
+      "and",
+      "@col1 = preceding-sibling::OP-RIGHT-PAREN/@col1 + 1",
+      "]"
+    )
+    matched_expressions <- xml2::xml_find_all(source_file$xml_parsed_content, xpath)
+    
+    lapply(matched_expressions, get_lint_from_expression, source_file = source_file)
+  })
+}
+
+get_lint_from_expression <- function(expression, source_file) {
+  expression <- xml2::as_list(expression)
+  Lint(
+    filename = source_file$filename,
+    message = "There should be a space between the right parenthesis and the function body.",
```

Comment:
I suggest the message should be `"There should be a space between right parenthesis and a body expression."`.

Reply:
Changed the message